### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by nuclearpinguin**
Break lines in README.md and fix double spaces and a typo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,19 +96,33 @@ make build
...
+    and `kedacore/keda-metrics-adapter` tagged with the tag you set in step 2
 4. If you haven't downloaded them before, clone the charts repository: `git clone git@github.com:kedacore/charts.git` 
-5. Still in terminal, navigate to the `chart/keda` folder (downlodaed in step 4), and run the following command (don't forget to replace the placeholder text in the command) `helm install . --set image.keda=kedacore/keda:$IMAGE_TAG,image.metricsAdapter=kedacore/keda-metrics-adapter:$IMAGE_TAG,image.pullPolicy=IfNotPresent`. This will use the images built at step 3. Notice the need to override the image pullPolicy to `IfNotPresent` in order to use the locally built images and not try to pull the images from remote repo on Docker Hub (and complain about not finding them).
-6. Once the keda operator pod is up, check the logs of both containers to verify everything running ok, eg: `kubectl logs <keda operator pod name> -c keda-operator-metrics-apiserver` and `kubectl logs <keda operator pod name> -c keda-operator`
+5. Still in terminal, navigate to the `charts/keda` folder (downlodaed in step 4), and run the following command
```
Comment (nuclearpinguin) [inline README.md]: Here was a typo `chart/keda` -> `charts/kead`

Comment (marchmallow) [reply README.md]: Thanks for fixing.. just noticed also this typo: `(downlodaed`

Comment (nuclearpinguin) [reply README.md]: Good catch! Fixed :)

**Comment by tomkerkhove** [comment]
It looks like the listing is now broken, can you please address that?

**Comment by nuclearpinguin** [comment]
> It looks like the listing is now broken, can you please address that?

Fixed :)

**Comment by ahmelsayed** [comment]
Thank you @nuclearpinguin!

**Pull request merged**